README.md

```diff
@@ -3,17 +3,72 @@
+1. Install **VcXsrv**
+    - If you are using Chocolatey: `choco install vcxsrv`
+    - Otherwise, download from [Source Forge](https://www.google.com/url?q=https%3A%2F%2Fsourceforge.net%2Fprojects%2Fvcxsrv%2F&sa=D&sntz=1&usg=AFQjCNFZLtdIHK50nhjd-rYEgBPhHWYt4A)
+2. Run **XLaunch** and choose the settings shown on the [MAAV Website](https://sites.google.com/umich.edu/maav/team/software/docker/guis-on-docker?pli=1&authuser=1) or [this article](https://dev.to/darksmile92/run-gui-app-in-linux-docker-container-on-windows-host-4kde)
+3. Before clicking **Finish**, save the configuration file to one of the following locations:
+    - `%AppData%\Xming`
+    - `%UserProfile%\Desktop`
+    - `%UserProfile%`
+4. Open **PowerShell** and run `ipconfig` to determine IP address (look at [MAAV Website](https://sites.google.com/umich.edu/maav/team/software/docker/guis-on-docker?pli=1&authuser=1) for help)
+5. In the **docker-compose.yml** file, under environment, add the IP Address to the display variable:
+    - `DISPLAY=<IP Address>:0.0` where "\<IP Address>" is replaced with your IP Adress from step 4
+    - docker-compose.yml should look something like this now
+        ```yml
+        version: "2"
+
+        services:
+        ros-demo:
+            image: ros-demo
+            privileged: true
+            volumes:
+            # Mount the current directory do everything in /tutorial within docker
+            - .:/tutorial:rw
+            environment:
+            - DISPLAY=1.1.1.1:0.0 # Change 1.1.1.1 to your IP Address
+            network_mode: "host"
+            container_name: ros-demo
+            command: "/bin/bash --init-file scripts/source-ros.sh" #source ros automatically
+        ```
+6. Follow instructions in ROS Tutorial Steps below
+
+#### Suggested Improvement (Optional): Docker Toolbox + WSL = tmux
+As you are working through the ROS Tutorials, you will see that they suggest running processes in one terminal, then opening a new window to run other processes. In Docker, this can be troublesome. You can run a process in the background: e.g. `roscore &`, but this solution doesn't work for every situation. When you want to control the movement of the turtle in turtlesim using the arrow keys, you cannot run that in the background. A solution that I have found is to use WSL (Windows Subsystem for Linux) and tmux.
+
+1. [Install Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+2. [Set up Docker Toolbox for Windows Home 10 and WSL](https://medium.com/@joaoh82/setting-up-docker-toolbox-for-windows-home-10-and-wsl-to-work-perfectly-2fd34ed41d51)
+3. Install tmux in WSL
+    - `sudo apt-get install tmux`
+4. Follow the steps in ROS Tutorial below to run the docker container
+5. When inside docker, open tmux and run roscore in one window
+    - `tmux`
```

Comment:
As an FYI this is a great post on making TMUX more usable 

https://www.hamvocke.com/blog/a-guide-to-customizing-your-tmux-conf/

Then you just have to mount the .tmux.conf file to use the changes your docker container

Reply:
Also it might be useful to mention the other way to launch a new terminal. `docker exec -it <container_name> bash`

This would avoid people having to learn a new program. Although tmux is the best

Reply:
Implemented in **tmux Customization and docker exec for new windows** commit